store/modules/events.ts

```diff
@@ -15,7 +16,7 @@ export { CommunityEvent, CommunityEventType, WorldLocation }
 
 export default {
   state: {
-    activeTab: 0,
+    communityEventsShowed: 'upcoming',
```

Comment:
To highlight the fact that the store should know nothing about the view, I'd like to avoid using the verb "show". We can use `activeSet` for referring to which of `futureCommunityEvents` or `pastCommunityEvents` are the filters acting on. I think the name, in addition to the possible values are enough to convey the meaning.